### PR TITLE
Fix for unhandled FireEye ETP exception

### PR DESCRIPTION
**Pull request by tundisto**
## Status
Ready

## Related Issues
No issue opened - fixed by SA (me) during troubleshooting

## Description
Fixes an unhandled exception that gets thrown when authentication fails when testing the FireEye ETP integration instance.  It does this by wrapping the exception with str() in return_error_entry().  This unhandled exception most likely also gets thrown all throughout the code but I don't have a working ETP instance to test that with, so it's probably fixing the problem everywhere.

## Screenshots
Bad (original): 
image

Fixed:  
image

## Related PRs
N/A

## Required version of Demisto
4.x

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
N/A

## Additional changes
None

### CODE REVIEW

**Comment by Itay4** [comment]
@tundisto please merge from master and add changelog

**Comment by tundisto** [comment]
@Itay4 Assuming this is so, but to confirm, in addition to the changelog, am I also supposed to put something in Releases/LatestRelease?  I ask because the circleci check is failing on this.

**Comment by Itay4** [comment]
@tundisto please merge from master

**Comment by Itay4** [comment]
@kirbles19 please review the [changelog](https://github.com/demisto/content/blob/4c5634f792bea1b396bc399997b2468d910b4ac2/Integrations/FireEyeETP/CHANGELOG.md)

**Comment by tundisto** [comment]
Converted to package format and merged master.

**Comment by Itay4** [comment]
please note [the build](https://circleci.com/gh/demisto/content/31324) fails

**Comment by kirbles19** [comment]
@tundisto @Itay4 Done.
Please check my changes for technical accuracy.

**Comment by tundisto** [comment]
@kirbles19 -- the change isn't specific to the integration test but rather for exceptions thrown everywhere in the integration, though I found it when testing the integration.

**Comment by Itay4** [comment]
@kirbles19 would you like to update the changelog?

**Comment by tundisto** [comment]
@kirbles19 would you mind updating the changelog?

**Comment by tundisto** [comment]
@kirbles19 this is ready for you

**Comment by kirbles19** [comment]
@Itay4 Done, no changes required to the changelog, looks good.

File: Integrations/FireEyeETP/FireEyeETP.yml [hunk 1]
```diff
@@ -307,6 +307,7 @@ script:
         90 days old are not available.
   isfetch: true
   runonce: false
+  subtype: python3
```
Comment (Itay4) [inline Integrations/FireEyeETP/FireEyeETP.yml]: it should be python2

**Comment by tundisto** [comment]
@Itay4 I cannot push the changes to the fix-fireeye-etp-exception-handling branch.  It looks to be a permissions issue on the content repo.  I'd really like to get this thing over the finish line.

**Comment by tundisto** [comment]
@Itay4 I no longer have permissions to merge.  If you care to do that I think we can close this out

**Pull request merged**